bin/firecrown

```diff
@@ -49,6 +50,7 @@ def run(action, config, output):
     # we associate a unique id with each analysis
     # let's write that out with some extra info
     write_metadata(analysis_id, output, config)
+    write_statistics(analysis_id, output, stats)
```

Comment:
stats is currently undefined here if we use the run-cosmosis branch

Reply:
Ack. Good catch and thank you! Fixed!